corehq/apps/performance_sms/forms.py

```diff
@@ -106,3 +95,80 @@ def source_id(self):
         if self.config.template_variables:
             return self.config.template_variables[0].source_id
         return ''
+
```

Comment:
There are a couple `# todo: need to support multiple sources`s above here, can those be dropped now or does that refer to something else?